### PR TITLE
gateways: replace fw3 with fw4, translate all rules to nftables; treewide: switch to 22.03

### PR DESCRIPTION
**Pull request by spolack**
This is running on saarbruecker-gw now.

Interesting commands to troubleshoot:
```
fw4 print
nft list ruleset

```

### CODE REVIEW

File: roles/cfg_openwrt/templates/uplink_gateway/config/firewall.j2 [hunk 1]
```diff
@@ -5,6 +5,7 @@ config defaults
 	option output		ACCEPT
 	option forward		REJECT
 
+
```
Comment (PolynomialDivision) [inline roles/cfg_openwrt/templates/uplink_gateway/config/firewall.j2:8]: why?

Comment (spolack) [reply roles/cfg_openwrt/templates/uplink_gateway/config/firewall.j2:8]: I'll remove that blank new line

**Comment by spolack** [comment]
I already tested that change on magda-core which is ipq40xx and magda-ap1.

**Comment by spolack** [comment]
Let's wait until https://github.com/openwrt/openwrt/pull/10847 is merged. That will hopefully allow us to run the 8M/32M Devices with current OpenWRT

**Comment by PolynomialDivision** [comment]
I would propose to merge and just keep the ubiquiti xm devices to 21.02. No idea when and if that is going to be merged.

**Comment by spolack** [comment]
Well, then lets lets use snapshot for them.

**Review by PolynomialDivision — APPROVED** [review]

**Pull request merged**